support/Pipelines/Bbh/Ringdown.py

```diff
@@ -26,6 +26,17 @@
 
 RINGDOWN_INPUT_FILE_TEMPLATE = Path(__file__).parent / "Ringdown.yaml"
 
+# Resolution levels defined in terms of p-refinement
```

Comment:
Also make this change please to choose correct resolution when transitioning from inspiral to ringdown:

```diff
diff --git a/support/Pipelines/Bbh/Inspiral.yaml b/support/Pipelines/Bbh/Inspiral.yaml
index aaee72e559..efd740e2fb 100644
--- a/support/Pipelines/Bbh/Inspiral.yaml
+++ b/support/Pipelines/Bbh/Inspiral.yaml
@@ -10,8 +10,7 @@ Next:
   With:
     inspiral_input_file_path: __file__
     inspiral_run_dir: ./
-    refinement_level: {{ L }}
-    polynomial_order: {{ P }}
+    lev: {{ Lev }}
     pipeline_dir: {{ pipeline_dir }}
     scheduler: {{ scheduler | default("None") }}
     copy_executable: {{ copy_executable | default("None") }}
```